style.css

```diff
@@ -47,7 +47,7 @@ input {
   font-weight: 14px;
 }
 
-.add-btn {
+button {
```

Comment:
Try not to hang styles on plain HTML tags, it can lead to these styles being applied where they shouldn't be. Better yet, create a separate button class.